gettsim/taxes/eink_st.py

```diff
@@ -3,17 +3,17 @@
 from gettsim.typing import IntSeries
 
 
-def st_kein_kind_freib_tu(
-    zu_verst_eink_kein_kinderfreib_tu: FloatSeries,
+def st_kein_kinder_fb_tu(
```

Comment:
at first glance this looks like weird German ('kein Kinder'). Maybe consider `kinderfb`.

Reply:
Agreed, see just above.